hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -458,7 +452,40 @@ public AbfsRestOperation createPath(final String path, final boolean isFile, fin
     return op;
   }
 
+  public AbfsRestOperation createPathBlob(TracingContext tracingContext, final String path, final boolean isFile,
```

Comment:
it is using same headers that were defined before calling this method. Can they be reused.

Reply:
Done.